### ISSUE TITLE
API Documentation 'x' button is no-op

### ISSUE THREAD

**Issue by mr-c** [seqan/seqan#865]
Steps to reproduce:

1) Go to http://docs.seqan.de/seqan/2.0.0/
2) Click on SeqFileIn link
3) Click 'x' in left pane
4) Be redirected back to the main page instead of the left pane disappearing.

**Comment by esiragusa** [comment]
It seems to work under Firefox but not under Safari nor Chrome...

**Comment by esiragusa** [comment]
@rrahn I don't know who maintains the dox js... please reassign it in case.

**Comment by rrahn** [comment]
@mr-c Can you please specify your browser version? It seems to work on all my instances:
safari v8.0.3
Firefox v31.4.0
Chrome v 40.0.2214.115

Using Mac OS X v10.10.2

**Comment by mr-c** [comment]
Perhaps it is due to the target for the 'X' is quite small and is overlapped by the target for the top level of the documentation.
![screenshot 2015-02-25 at 14 35 47 - edited](https://cloud.githubusercontent.com/assets/1330696/6378835/d3548180-bcfb-11e4-8e80-65216cfd3ca0.png)
![screenshot 2015-02-25 at 14 41 19 - edited](https://cloud.githubusercontent.com/assets/1330696/6378927/7c1a043e-bcfc-11e4-8e3a-b9c3e0286b71.png)

```
Google Chrome	40.0.2214.114 (Official Build) 
Revision	9f86a7bbfc2fddc858b75b00dc57661850dd7dc0
Platform	6457.94.0 (Official Build) stable-channel link
Blink	537.36 (@7408c7855fa5235f63fb36835682e32901eca22c)
JavaScript	V8 3.30.33.16
Flash	16.0.0.305-r1
User Agent	Mozilla/5.0 (X11; CrOS x86_64 6457.94.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.114 Safari/537.36
```

**Comment by esiragusa** [comment]
Fixed in #1100 #1101. @mr-c thanks for reporting! :+1:

**Comment by mr-c** [comment]
Always happy to be helpful, thanks for the follow-up!

On Thu, Aug 13, 2015, 09:32 Enrico Siragusa <notifications@github.com>
wrote:

> Fixed in #1100 <https://github.com/seqan/seqan/pull/1100> #1101
> <https://github.com/seqan/seqan/pull/1101>. @mr-c
> <https://github.com/mr-c> thanks for reporting! [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/seqan/seqan/issues/865#issuecomment-130751920>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe
